drivers/hmis/app/models/hmis/ce/match/expression/client_field_map.rb

```diff
@@ -97,11 +97,24 @@ def open_enrollment_project_types_excluding_incomplete_field
 
     def open_referral_project_types_field
       {
-        query: ->(clients) { project_types_query(clients, Hmis::Ce::Referral.active, :target_project) },
+        query: ->(clients) do
+          # Get project types from CE referrals
+          ce_referrals_result = project_types_query(clients, Hmis::Ce::Referral.active, :target_project)
+
+          # Get project types from legacy referrals
+          legacy_referrals_scope = HmisExternalApis::AcHmis::ReferralHouseholdMember.joins(:postings).
+            merge(HmisExternalApis::AcHmis::ReferralPosting.active)
+          legacy_referrals_result = project_types_query(clients, legacy_referrals_scope, postings: :project)
+
+          # Merge results from both referral types
+          ce_referrals_result.merge(legacy_referrals_result) { |_key, values1, values2| (values1 + values2).uniq }
```

Comment:
is order significant?

```suggestion
          ce_referrals_result.merge(legacy_referrals_result) { |_key, values1, values2| (values1 + values2).uniq.sort }
```

Reply:
Thanks good point. It shouldn't matter for evaluation, but I could see it coming into play when attributes are compare to decided whether they have been changed (aka whether to store update event in CE match candidate events). I will add the sort!